tests/controls/spinctrltest.cpp

```diff
@@ -368,4 +368,28 @@ TEST_CASE_METHOD(SpinCtrlTestCase3, "SpinCtrl::SetValueInsideEventHandler", "[sp
 #endif // wxUSE_UIACTIONSIMULATOR
 }
 
+TEST_CASE_METHOD(SpinCtrlTestCase1, "SpinCtrl::Increment", "[spinctrl]")
+{
+#if wxUSE_UIACTIONSIMULATOR
+    m_spin->Create(wxTheApp->GetTopWindow(), wxID_ANY, "",
+        wxDefaultPosition, wxDefaultSize,
+        wxSP_ARROW_KEYS | wxSP_WRAP);
+
```

Comment:
Could you please add
```suggestion
    CHECK( m_spin->GetIncrement() == 1 );
```
here to check the initial value too?